Automatic Validation ended with:
> 2024-02-09T14:10:53.9127213Z ##[error] Manifest Error: Schema Error. [Schema validation failed. Error context: <root>[PackageUrl] Description: Failed to match regex specified by 'pattern' constraint. Error context: <root> Description: Failed to validate against schema associated with property name 'PackageUrl'. ] File: Aircall.Aircall.locale.en-US.yaml

(Automated response - build 720.)

--- a/manifests/a/Aircall/Aircall/3.1.8/Aircall.Aircall.installer.yaml
+++ b/manifests/a/Aircall/Aircall/3.1.8/Aircall.Aircall.installer.yaml
@@ -11,9 +11,8 @@ InstallModes:
 - silent
 - silentWithProgress
 InstallerSwitches:
-  Silent: /install /quiet /norestart
-  SilentWithProgress: /install /quiet /norestart
-  Custom: /install /quiet /norestart
+  Silent: /quiet /norestart
+  SilentWithProgress: /passive /norestart
 UpgradeBehavior: install
 ProductCode: '{4FCE0E0B-28E5-4F09-9215-5438D92E91DA}'
 Installers:

--- a/manifests/a/Aircall/Aircall/3.1.8/Aircall.Aircall.locale.en-US.yaml
+++ b/manifests/a/Aircall/Aircall/3.1.8/Aircall.Aircall.locale.en-US.yaml
@@ -10,7 +10,7 @@ PublisherSupportUrl: https://aircall.io/inbound-call-center-software/
 PrivacyUrl: https://aircall.io/privacy/
 Author: Aircall
 PackageName: Aircall
-PackageUrl: A flexible solution that fits the teams you have today (and the plans you have for tomorrow).
+PackageUrl: https://aircall.io/download/
 License: Proprietary
 Copyright: Copyright © 2023, Aircall
 ShortDescription: A flexible solution that fits the teams you have today (and the plans you have for tomorrow).

--- a/manifests/a/Aircall/Aircall/3.1.8/Aircall.Aircall.locale.en-US.yaml
+++ b/manifests/a/Aircall/Aircall/3.1.8/Aircall.Aircall.locale.en-US.yaml
@@ -14,6 +14,6 @@ PackageUrl: https://aircall.io/download/
 License: Proprietary
 Copyright: Copyright © 2023, Aircall
 ShortDescription: A flexible solution that fits the teams you have today (and the plans you have for tomorrow).
-ReleaseNotesUrl: https://support.aircall.io/hc/en-gb/sections/10393020321821-Release-Notes
+# ReleaseNotesUrl: 
 ManifestType: defaultLocale
 ManifestVersion: 1.6.0

